packages/rest/src/rest.ts

```diff
@@ -332,7 +370,7 @@ function makeActionName(op: Operation, name: string | undefined): string {
 const actionsKey = createStateSymbol("actions");
 export function $action(context: DecoratorContext, entity: Operation, name?: string) {
   // Generate the action name and add it as an operation path segment
-  const action = makeActionName(entity, name);
+  const action = `!!${makeActionName(entity, name)}`;
```

Comment:
That's true. It's also true of `getCollectionAction`. I would argue those are both bugs. For `@actionSeparator` to work, it needs to know where the separator needs to go. `@segmentSeparator` only needed this for `@collectionAction` because of the existing path logic works, but in separating it out, that magic no longer works.

Thoughts?

Reply:
Actually, @daviwil I just pushed a change that adds the segment with the sentinel but only puts the name in action, so `getAction` will no longer return the value with the sentinel. On the other hand, `getCollectionAction` will now return `actionName` instead of `foo/actionName`. I could preserve the old behavior despite the fact that the separator might be wrong. Thoughts?